tutorials/publish/index.md

```diff
@@ -349,6 +349,34 @@ It also should be noted that if your etcd storage is safe and channels have not
 
 Our recommendations are following
 - Your should run ```snet treasurer claim-expired``` each 1-3 days. We recommend automate it using `cron`.
-- You can run ```claim-all``` command when your want. For example once in serveral month.
+- You can run ```claim-all``` command when your want. For example once in several months.
 
-For more information about the `SNET MultiPartyEscrow` check this [link](https://github.com/singnet/dev-portal/tree/master/docs/all/mpe). 
+For more information about the `SNET MultiPartyEscrow` check this [link](https://dev.singularitynet.io/docs/all/mpe/mpe/). 
+
+## Step 11 (optional). Withdraw AGI tokens from MPE
+
+After the step 10, all the AGIs are in the MultiPartyEscrow Contract (MPE).
+
+If you need to transfer them to your wallet, you must run the `SNET CLI` withdraw command.
+
+For example, to withdraw 5 AGIs from MPE:
+
+```
+# !!! check current account balance
+snet account balance
+  account: 0xA6E06cF37110930D2906e6Ae70bA6224eDED917B
+  ETH: **********
+  AGI: 0
+  MPE: 5
+
```

Comment:
I'm not sure that we need to show output at all... We could simply say that we can check balance with "snet account balance" and we can withdraw with "snet account withdraw".....

But if we need to show this output I would propose to add "#" prefix and remove account:
```
snet account balance
# account: **********
# ETH: **********
# AGI: 0
# MPE: 5
```

Reply:
The outputs have been removed.